dataset_generator/fuzzing-script.py

```diff
@@ -5,7 +5,20 @@
 NUM = [str(random.randint(1, 100)) for _ in range(50)]
 exist_vars = []
```

Comment:
It seems like you're using a stack to track the variable values, but you probably don't need to track those at all.  Instead, you can use such a stack to track the available variables.